src/openvic-simulation/economy/production/ProductionType.hpp

```diff
@@ -76,6 +77,26 @@ namespace OpenVic {
 		bool parse_scripts(DefinitionManager const& definition_manager);
 
 	public:
+		constexpr bool get_is_farm_for_tech() const {
+			if (game_rules_manager.get_use_simple_farm_mine_logic()) {
+				return _is_farm;
+			}
+
+			return !_is_mine && _is_farm;	
```

Comment:
```suggestion
			return !_is_mine && _is_farm;
```
Stray tab character at the end of this line